config/owasp/suppressions.xml

```diff
@@ -103,4 +103,7 @@
   <suppress>
     <cve>CVE-2022-22950</cve>
   </suppress>
+  <suppress>
+    <cve>CVE-2016-1000027</cve>
```

Comment:
The suppression should be in a different PR with master so the master is always green and the PR branch just contains feature changes. I know you said you did this to get the PR build through. Ideally, we should not have this in the feature PR.